src/lib/sdk.ts

```diff
@@ -99,36 +135,43 @@ class FlutterSDK {
     }
   }
 
-  private async initDarkSdkHomeFromFlutter(flutterLookup: string) {
+  private async initDarkSdkHomeFromSearchPaths() {
     try {
-      let flutterPath: string;
+      const sdks = await this.findSdks();
+      if (sdks.length > 0) {
+        this._sdkHome = sdks[0].location;
+        await this.initFlutterCommandsFromSdkHome();
+      }
+    } catch(error) {
+      log(`Error configuring sdk from searchPaths: ${error.message}}`);
+    }
+  }
 
-      if (flutterLookup.length == 0) {
-        flutterPath = (await which('flutter')).trim();
+  private async initDartSdkHomeFromLocalFvm() {
+    try {
+      const fvmLocation = join(workspace.workspaceFolder.uri.replace(/^file:/, ''), '.fvm/flutter_sdk');
```

Comment:
We can use `Uri.parse(workspace.workspaceFolder.uri).fsPath` to get file system path. `Uri` import from coc.nvim.